setup.py

```diff
@@ -95,36 +113,27 @@ def is_64_bit_python():
     return sys.maxsize > (1 << 32)
 
 
-class CleanLibraries(CleanCommand):
-    @staticmethod
-    def remove_dynamic_library():
-        shutil.rmtree(LIBDDWAF_DOWNLOAD_DIR, True)
+class Library_Download:
+    name = None
+    download_dir = None
+    version = None
+    url_root = None
+    available_releases = None
+    expected_checksums = None
 
-    def run(self):
-        CleanLibraries.remove_dynamic_library()
-        CleanCommand.run(self)
-
-
-class LibDDWaf_Download(BuildPyCommand):
-    @staticmethod
-    def download_dynamic_library():
-        TRANSLATE_SUFFIX = {"Windows": ".dll", "Darwin": ".dylib", "Linux": ".so"}
-        AVAILABLE_RELEASES = {
-            "Windows": ["win32", "x64"],
-            "Darwin": ["arm64", "x86_64"],
-            "Linux": ["aarch64", "x86_64"],
-        }
-        SUFFIX = TRANSLATE_SUFFIX[CURRENT_OS]
+    @classmethod
+    def download_artifacts(cls):
+        suffixes = cls.get_suffixes(CURRENT_OS)
 
         # If the directory exists and it is not empty, assume the right files are there.
         # Use `python setup.py clean` to remove it.
-        if os.path.isdir(LIBDDWAF_DOWNLOAD_DIR) and len(os.listdir(LIBDDWAF_DOWNLOAD_DIR)):
+        if os.path.isdir(cls.download_dir) and len(os.listdir(cls.download_dir)):
```

Comment:
```suggestion
        if os.path.isdir(cls.download_dir) and os.listdir(cls.download_dir):
```

Reply:
Ah this was already coded with `len`. The diff was something else. Feel free to ignore.

Reply:
Made good sense to me.  Plus--touch the line, own the line, or something.